lib/DataBase/schema.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE
+  music_albums (
+    id INT GENERATED BY DEFAULT AS IDENTITY,
+    genre VARCHAR(255),
+    label VARCHAR(255),
+    on_spotify BOOLEAN,
+    publish_date DATE,
+    archived BOOLEAN,
+    genre_id INT,
+    PRIMARY KEY (id),
+    CONSTRAINT fk_genre_id FOREIGN KEY (genre_id) REFERENCES genres (id)
+  );
```

Comment:
Kindly also add `author_id` as a column and make it a foreign key referencing the `author` table as an `item` class also contains `author` as a property and we need to implement all properties of the parent class